src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -340,7 +345,7 @@ const useGroupInfo = (channelId = null) => {
       setIsLoadingInitChat(true);
       await handleAnonymousMessage(selectedUser, channelId, 'chat');
     } catch (e) {
-      console.log(e);
+      console.error(e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing `console.log(e)` to `console.error(e)` is a good practice as it correctly logs the error messages.
```diff
- console.log(e);
+ console.error(e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->